boards/st/stm32f4_disco/stm32f4_disco.dts

```diff
@@ -187,3 +174,39 @@ zephyr_udc0: &usbotg_fs {
 &vbat {
 	status = "okay";
 };
+
+&dma1 {
+	status = "okay";
+};
+
+&plli2s {
+	/* Minimize audio clock error (with i2s mck-enabled) for all of:
+	 * - 22.05kHz / 16b, 24b or 32b
+	 * - 44.1kHz  / 16b, 24b or 32b
+	 * - 88.2kHz  / 16b or 24b
+	 */
+	mul-n = <271>;
```

Comment:
It may be useful to also specifies that it takes same input as main PLL and hence this one should not be changed w/o checking impact on PLLI2S